config/webpack.config.js

```diff
@@ -245,41 +251,94 @@ module.exports = env => {
         fileName: 'file-manifest.json',
       }),
     );
-  } else {
+  }
+
+  // Optionally generate landing pages in the absence of a content build.
+  if (buildOptions.scaffold) {
     const landingPagePath = rootUrl =>
       path.join(outputPath, '../', rootUrl, 'index.html');
 
+    const inlineScripts = [
+      'incompatible-browser.js',
+      'record-event.js',
+      'static-page-widgets.js',
+    ].reduce(
+      (scripts, filename) => ({
+        ...scripts,
+        [filename]: fs.readFileSync(path.join('src/site/assets/js', filename)),
+      }),
+      {},
+    );
```

Comment:
This was necessary to translate from the invalid Liquid `include` template code. Explained more in the template below.